includes/REST/apple-news-get-published-state.php

```diff
@@ -16,6 +16,17 @@
 function get_published_state_response( $data ) {
 	$response = [];
 
+	// Ensure Apple News is first initialized.
+	if ( ! \Apple_News::is_initialized() ) {
+		return new WP_Error(
+			'apple_news_bad_operation',
+			__( 'You must enter your API information on the settings page before using Publish to Apple News.', 'apple-news' ),
+			[
+				'status' => 400,
+			]
+		);
+	}
+
```

Comment:
Is there any value in abstracting this into a helper since it's used in multiple places?

Reply:
That's a good point. 👍